docs/src/examples/components/Divider/Variations/DividerExampleFitted.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Divider } from '@stardust-ui/react'
+
+const DividerExampleFitted = () => (
```

Comment:
this example could be eliminated - as there is no Children API variation for this case